src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -191,6 +192,11 @@ void goto_analyzer_parse_optionst::get_command_line_options(optionst &options)
     options.set_option("show", true);
     options.set_option("general-analysis", true);
   }
+  else if(cmdline.isset("show-on-source"))
```

Comment:
´ oututPlease add the option to the `--help` output.

Reply:
Done